pib_api/flask/controller/assistant_controller.py

```diff
@@ -0,0 +1,32 @@
+from flask import abort, Blueprint, make_response
+
+from app import app
+from schema.assistant_schema import assistants_schema, assistant_schema
+from service import assistant_service
+from service.assistant_service import get_assistant_by_id
+
+bp = Blueprint('assistant_controller', __name__)
+
+
+@bp.route('', methods=['GET'])
+def get_all_assistants_settings():
+    try:
+        assistants = assistant_service.get_all_assistants()
+        assistant_dtos = assistants_schema.dump(assistants)
+    except Exception as e:
```

Comment:
changed

Reply:
we still need the try-except block around the 'dump' call.

Reply:
It's changed for now, but out of curiosity, what kind of exceptions are we expecting to see here? Even if we pass `None` values or random dictionaries, the dump call will simply return an empty `dict`.

Reply:
the marshamllow documentation states, that in case of an invalid object, a `ValidationError` is thrown ([https://marshmallow.readthedocs.io/en/stable/marshmallow.schema.html#marshmallow.schema.Schema.dump](url)). I never tried it out though...